securedrop/journalist_app/utils.py

```diff
@@ -111,6 +114,11 @@ def validate_user(
                 "Please wait at least {num} seconds before logging in again.",
                 period
             ).format(num=period)
+        elif isinstance(e, InvalidOTPSecretException):
```

Comment:
@lsd-cat I'd be interested in your perspective on this addition - it's the only case where we provide a non-generic login error message, which could be problematic.

Reply:
Seems reasonable to me. Unless the 60 seconds/5 attempts is something that admins are encouraged to change, which does not look the case, a potential attacker already knows what is the limit regardless of the error messages.